tests/test_core.py

```diff
@@ -182,6 +182,15 @@ def test_variable_unit(test_df):
     npt.assert_array_equal(test_df.variables(include_units=True), exp)
 
 
+def test_filter_empty_df():
+    # test for issue seen in #254
+    cols = ['model', 'scenario', 'region', 'variable', 'unit', 2005, 2010]
```

Comment:
suggest to use `pyam.IAMC_IDX + [2005, 2010]`

Reply:
fixed